Agoda.Frameworks.DB/StoredProc.cs

```diff
@@ -49,4 +49,19 @@ public virtual SpParameter[] GetParameters(TRequest parameters)
                 .ToArray();
         }
     }
+
+    public class IAmNotAStoredProc : IStoredProc
```

Comment:
This naming can be fairly confusing. Is there a way to describe what it is rather than what it is not? :)